### DISCUSSION TITLE
Uncaught SyntaxError: Cannot use 'import.meta' outside a module

### DISCUSSION THREAD

**Discussion by AndreinNguyen** [web-infra-dev/rspack#2268] [Q&A]
I do in my code

```
import {} from "wagmi";
```

This error appear when I import wagmi to my project
```
Uncaught SyntaxError: Cannot use 'import.meta' outside a module 
```

Can anyone fix it?

**Comment by h-a-n-a** [comment]
You might have to convert this into a bug report in the issue section and we will dig into your problem later then.
